pkgs/games/katago/default.nix

```diff
@@ -34,14 +34,14 @@ let
 
 in env.mkDerivation rec {
   pname = "katago";
-  version = "1.8.0";
+  version = "1.8.1";
   githash = "8ffda1fe05c69c67342365013b11225d443445e8";
```

Comment:
```
  githash = "73bc3e38b3490cbe00179c9c37f5385dfd60c6bc";
```

Reply:
Unfortunately, it doesn't seem like this particular package can be automatically updated because of this. :(